test/LaunchDarkly.EventSource.Tests/EventSourceEncodingTest.cs

```diff
@@ -52,26 +44,34 @@ private static string MakeLongString(int startNum, int endNum)
             return ret.ToString();
         }
 
+        private static Handler MakeStreamHandler(Encoding encoding)
+        {
+            var ret = Handlers.StartChunks("text/event-stream", encoding);
+            foreach (var chunk in streamChunks)
+            {
+                ret = ret.Then(Handlers.WriteChunkString(chunk, encoding));
+            }
+            return ret.Then(Handlers.Hang());
+        }
+
         [Theory]
         [InlineData(true)]
         [InlineData(false)]
         public void CanReceiveUtf8EventDataAsStrings(bool setExplicitEncoding)
```

Comment:
Most of the changes in this file are pretty straightforward and follow the same pattern as this test: instead of using a `StubMessageHandler` (which was a way of faking an HTTP client transaction without doing any I/O), it uses an `HttpHelpers.HttpServer` configured with some combination of `HttpHelpers` handlers. The changes look bigger than they are due to different indentation of the main block in each method. See PR comments for anything else worth noting.